### DISCUSSION TITLE
dark and light, different brand colors it's possible?

### DISCUSSION THREAD

**Discussion by tassoman** [vuejs/vitepress#2097] [Q&A] [answered]
Hello there, it's possible to override CSS variables, setting different colours for dark and light mode? My goal is to adapt link's colour contrast in relation to background page. 🤔

**Comment by alephpi** [comment]
overwrite .dark

**Reply by tassoman** [reply]
Please, can you explain more?

**Reply by Zhengqbbb** [reply, marked as answer]
[docs/.vitepress/theme/index.ts](https://github.com/Zhengqbbb/cz-git/blob/5277b1a820d5decb36f5bd736ec469a0e88b33f4/docs/.vitepress/theme/index.ts#L6)
[docs/.vitepress/theme/style/vars.css](https://github.com/Zhengqbbb/cz-git/blob/5277b1a820d5decb36f5bd736ec469a0e88b33f4/docs/.vitepress/theme/style/vars.css#L54)

**Reply by tassoman** [reply]
Clear example, thanks.
